AppManager/app_manager_server.py

```diff
@@ -414,7 +400,8 @@ def unmonitor_and_terminate(watch):
   except OSError:
     logging.error("Error deleting {0}".format(monit_config_file))
 
-  IOLoop.current().spawn_callback(kill_instance, watch, instance_pid)
```

Comment:
so then the function kill_instance isn't needed anymore right?

Reply:
Good point. No, it is not.